src/main/resources/springintegration/flows.xml

```diff
@@ -7,11 +7,6 @@
   http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd">
 
     <int:annotation-config/>
-	<import resource="case-outbound-flow.xml"/>
-    <import resource="collex-event-messager-outbound.xml"/>
-    <import resource="collex-event-messager-inbound.xml"/>
```

Comment:
are we able to delete these files as well as removing them from here?

Reply:
yes.. don't know why my last commit didn't fo in.. I did remove it but did not check in.

Reply:
done